packages/components/src/DatePicker/DatePicker.css

```diff
@@ -0,0 +1,97 @@
+.datePickerParentWrapper {
```

Comment:
Just a note: Don't need to act upon it. 

You don't have to make this as specific as possible as it won't clash with other classnames since those gets automagically adjusted to not class by css modules. Can just say wrapper.